.gitignore

```diff
@@ -0,0 +1,5 @@
+debug/
+target/
+**/*.rs.bk
+
+.idea/
```

Comment:
Why not?

Reply:
Because that is specific to your development environment and should be contained either in your global .gitignore file or in your local repositories .git/info/exclude file.
At least that's the way I was _taught_ :)

But as I said, super-nit, feel free to leave as is.

Reply:
The reason I always include this is because I don't want anyone checking in their .idea even if they don't have it in their global .gitignore.

Reply:
Following that argument we should also include .eclipse, ~ files for vim, whatever vscode does ..

I totally understand where you are coming from and please, absolutely do merge like this!! This is really more of a philosophical discussion :)